coaster/utils.py

```diff
@@ -31,11 +34,23 @@
 
 # --- Common delimiters and punctuation ---------------------------------------
 
-_strip_re = re.compile(ur'[\'"`‘’“”′″‴]+')
-_punctuation_re = re.compile(ur'[\t +!#$%&()*\-/<=>?@\[\\\]^_{|}:;,.…‒–—―«»]+')
-_username_valid_re = re.compile('^[a-z0-9]([a-z0-9-]*[a-z0-9])?$')
-_ipv4_re = re.compile('^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$')
-_tag_re = re.compile('<.*?>')
+def strip_re_py3():
+    return r'[\'"`‘’“”′″‴]+'
+
+def strip_re_py2():
+    return six.text_type(r'[\'"`‘’“”′″‴]+', 'utf-8')
+
+def punctuation_re_py3():
+    return r'[\t +!#$%&()*\-/<=>?@\[\\\]^_{|}:;,.…‒–—―«»]+'
+
+def punctuation_re_py2():
+    return six.text_type(r'[\t +!#$%&()*\-/<=>?@\[\\\]^_{|}:;,.…‒–—―«»]+', 'utf-8')
+
+_strip_re = re.compile(strip_re_py3() if six.PY3 else strip_re_py2())
+_punctuation_re = re.compile(punctuation_re_py3() if six.PY3 else punctuation_re_py2())
+_username_valid_re = re.compile(str('^[a-z0-9]([a-z0-9-]*[a-z0-9])?$'))
+_ipv4_re = re.compile(str('^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$'))
+_tag_re = re.compile(str('<.*?>'))
```

Comment:
Switched it to lambda function, but without using `ur''` as that still throws error. The code is a little bit cleaner now.